### PR TITLE
Adapt black v23.1.0.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ repos:
         exclude: conda/meta.yaml
  
   - repo: https://github.com/psf/black
-    rev: 22.12.0
+    rev: 23.1.0
```
Comment (wvreeven) [inline .pre-commit-config.yaml:9]: Please also update `pre-commit-hooks` to `v4.4.0` and isort to `5.12.0`.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 - qt5-qtbase-devel (install by `yum`)
 - xorg-x11-server-Xvfb (optional, install by `yum`)
 - qasync (install by `conda -c conda-forge`)
-- [black](https://github.com/psf/black) (22.12.0, optional)
+- [black](https://github.com/psf/black) (23.1.0, optional)
 - [flake8](https://github.com/PyCQA/flake8) (4.0.1, optional)
 - [isort](https://github.com/PyCQA/isort) (5.10.1, optional)
```
Comment (wvreeven) [inline README.md:20]: Please update  to `5.12.0`.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please also update `pre-commit-hooks` to `v4.4.0` and isort to `5.12.0`.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (teweitsai) [reply .pre-commit-config.yaml:9]: Done in the commit: [ad8e2e3](https://github.com/lsst-ts/ts_m2gui/pull/34/commits/ad8e2e32b04894e7bf312394b459378a4110f482). Thanks!

File: README.md (hunk 2 above)
Comment (teweitsai) [reply README.md]: Done in the commit: [ad8e2e3](https://github.com/lsst-ts/ts_m2gui/pull/34/commits/ad8e2e32b04894e7bf312394b459378a4110f482). Thanks!

**Pull request merged**